### PR TITLE
Actionpack typo fixes.

### PR DESCRIPTION
**Pull request by alkesh26**

### CODE REVIEW

File: actionpack/lib/abstract_controller/rendering.rb [hunk 1]
```diff
@@ -7,7 +7,11 @@
 
 module AbstractController
   class DoubleRenderError < Error
-    DEFAULT_MESSAGE = "Render and/or redirect were called multiple times in this action. Please note that you may only call render OR redirect, and at most once per action. Also note that neither redirect nor render terminate execution of the action, so if you want to exit an action after redirecting, you need to do something like \"redirect_to(...) and return\"."
+    DEFAULT_MESSAGE = "Render and/or redirect were called multiple times in this action. " \
```
Comment (gmcgibbon) [inline actionpack/lib/abstract_controller/rendering.rb]: For breaking up strings, you could also use a heredoc and [squish](https://edgeapi.rubyonrails.org/classes/String.html#method-i-squish).

**Review by gmcgibbon — APPROVED** [review]

File: actionpack/lib/abstract_controller/rendering.rb (hunk 1 above)
Comment (kaspth) [reply actionpack/lib/abstract_controller/rendering.rb]: These are cosmetic changes which we don’t accept.

File: actionpack/lib/action_controller/renderer.rb [hunk 2]
```diff
@@ -73,7 +73,7 @@ def initialize(controller, env, defaults)
     # The primary options are:
     # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
     # * <tt>:file</tt> - Renders an explicit template file. Add <tt>:locals</tt> to pass in, if so desired.
-    #   It shouldn’t be used directly with unsanitized user input due to lack of validation.
+    #   It shouldn't be used directly with unsanitized user input due to lack of validation.
```
Comment (kaspth) [inline actionpack/lib/action_controller/renderer.rb]: Is this really necessary?

File: actionpack/lib/action_controller/metal/strong_parameters.rb [hunk 3]
```diff
@@ -795,7 +795,7 @@ def init_with(coder) # :nodoc:
         @permitted  = coder.map["ivars"][:@permitted]
       when "!ruby/object:ActionController::Parameters"
         # YAML's Object format. Only needed because of the format
-        # backwardscompability above, otherwise equivalent to YAML's initialization.
+        # backwards compatibility above, otherwise equivalent to YAML's initialization.
```
Comment (kaspth) [inline actionpack/lib/action_controller/metal/strong_parameters.rb]: I’d rather keep the git history here.

File: actionpack/lib/abstract_controller/rendering.rb (hunk 1 above)
Comment (rafaelfranca) [reply actionpack/lib/abstract_controller/rendering.rb]: Yup. Can you please revert. If you want to fix typos, it is fine. If you want to change the way the code looks, please don't.

File: actionpack/lib/action_controller/metal/strong_parameters.rb (hunk 3 above)
Comment (rafaelfranca) [reply actionpack/lib/action_controller/metal/strong_parameters.rb]: I think it is fine here. That word doesn't exist 😄

**Comment by alkesh26** [comment]
@kaspth @rafaelfranca I think this is good to go now. We can merge this.

**Pull request merged**